ReactWindows/ReactNative.Shared.Tests/Internal/MockJavaScriptExecutor.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using System;
+
+namespace ReactNative.Tests
+{
+    class MockJavaScriptExecutor : IJavaScriptExecutor
+    {
+        public Func<string, string, JArray, JToken> OnCallFunctionReturnFlushedQueue { get; set; } = EmptyCallFunctionReturnFlushedQueue;
+        public Func<int, JArray, JToken> OnInvokeCallbackAndReturnFlushedQueue { get; set; } = EmptyInvokeCallbackAndReturnFlushedQueue;
+        public Func<JToken> OnFlushQueue { get; set; } = EmptyFlushQueue;
+
+        public Action<string, string> OnRunScript { get; set; } = EmptyRunScript;
+        public Action<string, JToken> OnSetGlobalVariable { get; set; } = EmptySetGlobalVariable;
+        public Action OnDispose { get; set; } = EmptyAction;
+
+        private static readonly Action EmptyAction = () => { };
+        private static readonly Action<string, string> EmptyRunScript = (_, __) => { };
+        private static readonly Action<string, JToken> EmptySetGlobalVariable = (_, __) => { };
+        private static readonly Func<string, string, JArray, JToken> EmptyCallFunctionReturnFlushedQueue = (_, __, ___) => { throw new NotImplementedException(); };
+        private static readonly Func<int, JArray, JToken> EmptyInvokeCallbackAndReturnFlushedQueue = (_, __) => { throw new NotImplementedException(); };
+        private static readonly Func<JToken> EmptyFlushQueue = () => { throw new NotImplementedException(); };
+
```

Comment:
Probably wrong in the original file, but static initializers should probably come before property initializers (would be good to make it even more explicit that these definitely run before the property initializers).